contrib/centos/aziot-identity-service.spec

```diff
@@ -185,6 +182,8 @@ fi
 %doc THIRD-PARTY-NOTICES
 %license LICENSE
 
+%exclude /root/rpmbuild/BUILD/aziot-identity-service-%{version}/target/x86_64-unknown-linux-gnu/release/build/
```

Comment:
This assumes the buildroot is `/root/rpmbuild`, which it doesn't have to be if someone else is using the spec file to build the package.

Also you can use `%{name}-%{version}` instead of `aziot-identity-service-%{version}`

Reply:
I've played around with a whole bunch of prefixes (%{buildroot}, %{builddir}, both), but none of them seemed to work. I can keep iterating, if you've got another suggestion.

Reply:
`%exclude %{_builddir}/%{name}-%{version}/target/x86_64-unknown-linux-gnu/release/build/`

works for me. Just tried it locally.

Reply:
The CI hath disagreed

Reply:
Yes, as I told you on Teams already :)